garrysmod/lua/includes/util/color.lua

```diff
@@ -22,51 +78,51 @@ end
 -----------------------------------------------------------]]
 function ColorAlpha( c, a )
 
-	return Color( c.r, c.g, c.b, a )
-	
+	c.a = a

```

Comment:
Here's an example in GMod itself:

https://github.com/garrynewman/garrysmod/blob/3e138636eb1b0ad6ed785dedf350020755cff5f1/garrysmod/lua/entities/widget_base.lua#L205-L208

The later call will be transparent with this change while before it would not be.